### ISSUE TITLE
Unknown order for properties files with same name

### ISSUE THREAD

**Issue by lightZebra** [scalecube/scalecube-config#149]
If we have 2 or more config.properties files with same name and key and different value we don't know  which one would be applied.

**Comment by artem-v** [comment]
Makes sense to look at https://github.com/scalecube/scalecube-config/blob/develop/config/src/main/java/io/scalecube/config/source/ClassPathConfigSource.java#L103
